Source/SwiftLintFramework/Rules/Style/SortedImportsRule.swift

```diff
@@ -173,6 +108,24 @@ struct SortedImportsRule: CorrectableRule, ConfigurationProviderRule, OptInRule
         }
     }
 
+    /// - returns: whether `lhs` should come before `rhs` based on a comparison of the contents of the import lines
+    private func should(_ lhs: Line, comeBefore rhs: Line) -> Bool {
+        switch configuration.grouping {
+        case .attributes:
+            let lhsAttributes = lhs.importAttributes()
+            let rhsAttributes = rhs.importAttributes()
+            if lhsAttributes != rhsAttributes {
+                if lhsAttributes.isEmpty != rhsAttributes.isEmpty {
+                    return rhsAttributes.isEmpty
+                }
+                return lhsAttributes < rhsAttributes
+            }
+            return lhs.importModule().lowercased() <= rhs.importModule().lowercased()
+        case .name:
+            return lhs.importModule().lowercased() <= rhs.importModule().lowercased()
```

Comment:
This line is duplicated and should therefore be moved outside the `switch` statement.